src/sdk/flyover.ts

```diff
@@ -484,10 +490,30 @@ export class Flyover implements Bridge {
     return isPegoutRefundable(this.getFlyoverContext(), quote)
   }
 
-  async refundPeginQuote (_quote: Quote, _providerSignature: string, _userBtcTransactionHash: string): Promise<string> {
-    const randomBytes = crypto.randomBytes(32)
+  /**
```

Comment:
remove the mention to refundPegout

Reply:
also I'm not sure about the naming of this (not sure if we discussed it or not) but refundPegin might be confusing as it is not an actual refund (because the user gets the token in the destination currency). Perhaps it would be better to name it as registerPegin in sdk and the if the UI wants to present as refund is up to them?